File: uk.co.norphos.crystallography.toolkit/src/uk/co/norphos/crystallography/toolkit/Crystal.java
package uk.co.norphos.crystallography.toolkit;

import uk.co.norphos.crystallography.api.CrystalSystem;
import uk.co.norphos.crystallography.api.Lattice;

public class Crystal {
	
	private UnitCell unitCell;
	private CrystalSystem crystalSystem;
	
	public Crystal(Lattice lattice) {
		unitCell = new UnitCell(lattice);
	}
	
	public UnitCell getUnitCell() {
		return unitCell;
	}

}

File: pynorcryst/pynorcrysttk/crystal.py
from unit_cell import UnitCell

class Crystal(object):

    def __init__(self, lattice, spacegroup=None):
        self.unit_cell = UnitCell(lattice)

        #TODO This is not correct, this is a temporary fudge
        self.crystal_system = self.unit_cell.crystal_family
